src/components/LobbyErrorDialog.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <BaseDialog
+    id="ErrorDialog"
+    :model-value="showLobbyError"
+    title="Oops!"
+    :opacity="1"
+    data-cy="create-game-dialog"
+  >
+    <template #body>
+      <div>
+        <h4>Cannot Join Lobby {{ errorMessage }}</h4>
```

Comment:
Semantically this is more of a paragraph than a header, so I think it makes sense to put this in a `<p>` tag and style it however necessary to make it look right